.gitignore

```diff
@@ -43,5 +43,5 @@ out/
 
 # 1BRC
 /measurements*.txt
-*.out
+/*.out
```

Comment:
Not sure why this was ignored, changed to ignore only in the repo root

Reply:
I use these in the eval tooling to verify the expected output. Just ignoring in the root is fine.